Client/src/main/java/edu/sdccd/cisc191/template/VegetableGUI.java

```diff
@@ -58,9 +58,7 @@ public VegetableGUI(Client viewPlantManagementUI) {
 
         Button addButton = new Button("Add");
         addButton.setPrefWidth(150);
-        addButton.setOnAction(e -> {
-            addVegetable();
-        });
+        addButton.setOnAction(e -> addVegetable());
```

Comment:
Change from Statement lambda to Expression lambda just to make it look more clean.